data.h

```diff
@@ -0,0 +1,45 @@
+#include <iostream>
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "cryptopp/sha.h"
+#include "cryptopp/Integer.h"
+
+class Data {
+  public:
+    const static unsigned int kBlockSize = 16;
+    const static unsigned int kDigestSize = 32;
+
+    // Empty constructor
+    Data();
+    // Converts byte array into Data object
+    Data(byte* data, size_t sizeIn = kBlockSize);
+    // Converts Integer into Data object
+    Data(IROHA_CRYPTO_HH::Integer data, unsigned int size = kBlockSize);
+    ~Data();
+
+    // Encodes byte array of object into base64 string
+    std::string toString();
```

Comment:
imo it's better to make it `static std::string to_string(const Data&)`, just as in stl

Reply:
Alright :+1: